### ISSUE TITLE
clarification on the current state of the module

### ISSUE THREAD

**Issue by KeLaiFu** [L71/BleepDrum-Eurorack#1]
First of all, thanks for sharing your work, my rack is currently sporting 2 of your MS20 modules.

Apologies if I am missing the obvious, but in the readme it states: 
'The internal drum sequencing in the module works I think - I have no original Bleep Drum to compare with - but I can't seem to get it to record or play back anything at all when the circuit sits on a breadboard (this is also why a clock output is missing). It works fine as a sound source...'

Does this mean the final PCBs also have issues recording and playing back, or it was something you experienced while breadboarding?

I would like to build this myself and I am not concerned about the internal recoding functions anyway, but if the record controls arent usuable, I will remove them from the design. 

Thanks again

**Comment by BleepLabs** [comment]
Hello. I'm not involved with this project at all but most probably there's an issue with pin assignments as I've changed those a few times over the years. What version of the code is this hardware intended for? Looking at the schematics it doesn't line up with the current version. That would explain why recording isn't working.

**Comment by L71** [comment]
Hi! 
@KeLaiFu  Cool! Happy to hear that!   =)  
Re: the bleep drum module, if I remember correctly sequence recording actually works on the finished module but I have never used it like that for more than a few minutes so the play/record functions may well be a bit broken without me realising it. I have no modular things connected at the moment so I can't check it.

@BleepLabs  Hi! :). Hmm, I wasn't aware that pin assignments had changed. It appears I used firmware v12 when building this if the README is correct.

**Comment by KeLaiFu** [comment]
> Hi! @KeLaiFu Cool! Happy to hear that! =) Re: the bleep drum module, if I remember correctly sequence recording actually works on the finished module but I have never used it like that for more than a few minutes so the play/record functions may well be a bit broken without me realising it. I have no modular things connected at the moment so I can't check it.
> 

That's fine for me, I thought about this module over the last few days and decided I wanted to make it look somewhat like a large version of the original, with illuminated arcade style buttons dominating the panel, and omitting the record functions anyway, so as long as it can be triggered via jacks and buttons, I am good.  Thanks for the feedback

**Comment by BleepLabs** [comment]
Yeah it's a little confusing. Use this code and all the buttons should work. https://github.com/BleepLabs/Bleep-Drum/tree/master/BLEEP_DRUM_15_legacy_upgrade
